setup/homestead.rst

```diff
@@ -50,9 +50,11 @@ configuration:
     sites:
         - map: symfony-demo.dev
           to: /home/vagrant/projects/symfony_demo/web
-          type: symfony
+          type: symfony4
 
-The ``type`` option tells Homestead to use the Symfony nginx configuration.
+The ``type`` option tells Homestead to use the Symfony nginx configuration. 
+Homestead now supports a Symfony2 web layout with app.php and app_dev.php when
+using type: symfony2 and an index.php layout when using type: symfony4.
```

Comment:
When `symfony` is used it defaults to `serve_symfony2.sh` configuration (this hasn't changed). I added the symfony2 for clarity when discussing the new `symfony4` type.